src/test/java/com/google/ehub/servlets/ProfileServletTest.java

```diff
@@ -149,6 +149,57 @@ public void postResquestWithFalseEditParam_SendsRedirect() throws IOException {
     verify(response).sendRedirect(REDIRECT);
   }
 
+  @Test
+  public void postResquestWithTrueEditParam_SendsRedirect() throws IOException {
+    Entity userEntity = new Entity(PROFILE_ITEM_KIND);
+
+    userEntity.setProperty(NAME_PROPERTY_KEY, NAME);
+    userEntity.setProperty(EMAIL_PROPERTY_KEY, EMAIL);
+    userEntity.setProperty(USERNAME_PROPERTY_KEY, USERNAME);
+    userEntity.setProperty(BIO_PROPERTY_KEY, BIO);
+
+    datastoreService.put(userEntity);
```

Comment:
You could also use the same function to create an entity and put it in Datastore right here.